src/BusinessLayer/TelemetryReporting.h

```diff
@@ -31,8 +39,12 @@ class TelemetryReporting
 
    void writeFloatIntoData(unsigned char* data, int index, const float& value);
    void sendData(const unsigned char* data, int length);
+   void connectToView();
+   void sendCmuData(unsigned char cmuDataIndex);
+   void sendMpptData(unsigned char mpptDataIndex);

```

Comment:
These don't really need to be private functions, it's making the header file quite complicated. Just `sendMpptData` which loops through all indexes will suffice.